contracts/truefi/TrueFiPool.sol

```diff
@@ -75,6 +75,8 @@ contract TrueFiPool is ITrueFiPool, IPauseableContract, ERC20, ReentrancyGuard,
 
     ITrueLender2 public _lender2;
 
+    address public safu;
```

Comment:
Why we need safu pointer here if we are not about to use it?

Reply:
We should add liquidate function same as to Pool2

Reply:
Didn't have time to do this yet